build/packer/ansible/roles/docker-setup/templates/pmm2.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=PMM2 Docker container
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+ExecStartPre=-/usr/bin/docker exec pmm-server stop
+ExecStartPre=-/usr/bin/docker rm pmm-server
+ExecStart=/usr/bin/docker run --volume /srv/:/srv/ --rm --name pmm-server \
```

Comment:
Let's use a volume instead of a data container. We are refactoring it to be the same everywhere.